pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -607,14 +624,18 @@ func selectVictimsOnNode(
 		if corev1helpers.PodPriority(p.Pod) < podPriority {
 			potentialVictims = append(potentialVictims, p.Pod)
 			if err := removePod(p.Pod); err != nil {
-				return nil, 0, false
+				message := fmt.Sprintf("Failed to remove victim pod %v on node %v: %v", p.Pod.Name, nodeInfo.Node().Name, err.Error())
```

Comment:
`RunPreFilterExtensionRemovePod` seems to have already returned verbosed error info, I'd suggest simplifying the final message.

Reply:
yes, that looks more neat.